README.md

```diff
@@ -16,15 +18,25 @@ To install, follow the [installation instructions for your Operating System](htt
 
 1. Get local copies of the project submodules so you can build and run your site locally:
    - `git submodule update --init --recursive`
-
-2. Build your site:
+1. Build your site:
    - `hugo server`
    - If there are any errors, check that you have the latest version of Hugo, and are using the extended version.
+1. Preview your site in your browser at: http://localhost:1313/
+1. As you make changes to the site, your browser will automatically reload your local dev site so you can easily and quickly see your changes. 
 
-3. Preview your site in your browser at: http://localhost:1313/
+## Optional Link Checking
```

Comment:
Want to make sure that it's clear that _local_ link checking is optional... Is there a better way to bring in the optional sub header just before the install instructions?

```suggestion
## Link Checking [Optional]
```

Reply:
I feel like given this document is entirely about doing dev, that's it's obvious you're doing link checking against local host. We don't say that you install `hugo` to do local dev, nor call out that you're building docs to do local dev, but both cite `localhost:1313`

That said, I don't feel strongly about this ;)  If you think we need to change some language here to call this out, I'm game.